.conda/meta.yaml

```diff
@@ -1,20 +1,19 @@
 ###############################################################################
-## Fichier de description du package pour Anaconda.org
-## Attention, les chaines PYPI_VERSION, PYPI_URL et PYPI_SHA256 sont remplacées
-## par la CI, il faut les conserver.
+## File describing the package for Anaconda.org
+## It uses Jinja2 templating code to retreive information from setup.py
 ###############################################################################
 
 {% set name = "OpenFisca-Survey-Manager" %}
-{% set version = "PYPI_VERSION" %}
+{% set data = load_setup_py_data() %}
+{% set version = data.get('version') %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 # openfisca-survey-manager package source
 source:
-  url: PYPI_URL
-  sha256: PYPI_SHA256
+  path: ..
```

Comment:
Using repository sources instead of PyPI:
➕ avoids conda packaging issues when PyPI timeouts (takes some time to publish the package)
➕ allows for conda package building tests (locally or on working branch) when `.conda/meta.yaml` is edited (as it now linked to the `setup.py` file that is included in the build)
➖ PyPI and conda packages are built separately and as such, could be slightly different. But considered here as unlikely to occur.